packages/flutter/test/material/progress_indicator_test.dart

```diff
@@ -1019,7 +1035,29 @@ void main() {
 
     await expectLater(
       await animationSheet.collate(20),
-      matchesGoldenFile('material.circular_progress_indicator.indeterminate.png'),
+      matchesGoldenFile('material.m2_circular_progress_indicator.indeterminate.png'),
+    );
+  }, skip: isBrowser); // https://github.com/flutter/flutter/issues/56001
+
+  testWidgets('Material3 - Indeterminate CircularProgressIndicator uses expected animation', (WidgetTester tester) async {
+    final AnimationSheetBuilder animationSheet = AnimationSheetBuilder(frameSize: const Size(40, 40));
+
+    await tester.pumpFrames(animationSheet.record(
+      Theme(
+        data: ThemeData(useMaterial3: true),
+        child: const Directionality(
+          textDirection: TextDirection.ltr,
+          child: Padding(
+            padding: EdgeInsets.all(4),
+            child: CircularProgressIndicator(),
+          ),
+        ),
+      ),
+    ), const Duration(seconds: 2));
+
+    await expectLater(
+      await animationSheet.collate(20),
+      matchesGoldenFile('material.m3_circular_progress_indicator.indeterminate.png'),
```

Comment:
Above 
` matchesGoldenFile('m3_material.border_paint_below.png'),` 

but here it's `material.m3`

Reply:
Just updated! Thanks for helping review!